client/src/components/Upload/index.js

```diff
@@ -12,23 +17,39 @@ class Upload extends Component {
   handleUploadFile = (event) => {
     const img = new FormData();
     img.append('file', event.target.files[0]);
-    axios.post('/add-to-amazon', img)
-      .then(({ data }) => {
-        this.setState({ details: data });
-        this.props.history.push({ pathname: '/get-details', details: this.state.details });
-      });
+    axios.post('/add-to-amazon', img).then(({ data }) => {
+      this.setState({ details: data });
+      this.props.history.push({ pathname: '/get-details', details: this.state.details });
+    });
   };
 
   render() {
     return (
       <React.Fragment>
         <Title />
         <Header />
-        <form>
-          <input label="upload file" type="file" onChange={this.handleUploadFile} />
-          <br />
-          <image id="logo" src={this.state.loc} alt="uploaded photo" height="300" width="300" />
-        </form>
+        <Instructions>
+          <TakeGoodPhoto>
```

Comment:
Have a look at the style in Figma: the text is currently too large, it can go down a few pixels.